### DISCUSSION TITLE
Screen - Please add a "Set default Decoder Quality for RAW"

### DISCUSSION THREAD

**Discussion by QuakeDave** [videovillage/community#198] [Ideas]
Dear Videovillage Team,

thanks for screen, it is a great product!

Please add an option to set a default decoder quality for RAW files. Especially because it is not possible in a playlist to sync/apply current preferences of the decoder quality for all clips in the playlist. I won't even save the decoder quality when saved as a playlist.

The only option is to manually click on every RAW video, every time you open the software... I mainly bought it for going through my RAW footage.

Other than that I am really happy with the software! :-)

Best regards!

**Comment by SteveOakley123** [comment]
1000% where are the fixes for settings across multiple clips  ??? LUTS, PB settings like RAW decode quality. This has to be the most requested thing to fix....
